docs/mutations.md

```diff
@@ -81,6 +81,15 @@ def resolve_logout(_, info):
     ...
 ```
 
+### Binding Mutation Resolvers
```

Comment:
Can you change this section into quote? That will give it yellow background that we are using to accent important bits.

Reply:
✅